include/unifex/let_value.hpp

```diff
@@ -438,25 +433,80 @@ class _sender<Predecessor, SuccessorFactory>::type {
 
     return std::max(pred(), std::min(succ(), blocking_kind::maybe()));
   }
+
+  friend instruction_ptr
+  tag_invoke(tag_t<get_return_address>, const type& t) noexcept {
+    return t.returnAddress_;
+  }
 };
 
 namespace _cpo {
 struct _fn {
+private:
+  struct _impl_fn {
+    template(typename Predecessor, typename SuccessorFactory)(
+        requires tag_invocable<_fn, Predecessor, SuccessorFactory>) auto
```

Comment:
Can you sneak some // comments in here to trick clang-format into making this slightly nicer, please?
```suggestion
    template(typename Predecessor, typename SuccessorFactory) //
    (requires tag_invocable<_fn, Predecessor, SuccessorFactory>) //
    auto
```